connectors/netty-connector/src/main/java/org/glassfish/jersey/netty/connector/NettyConnector.java

```diff
@@ -312,6 +312,14 @@ private String buildPathWithQueryParameters(URI requestUri) {
         }
     }
 
+    private String buildPathWithQueryParameters(URI requestUri) {
```

Comment:
When I do "view the file", it has:
```
private String buildPathWithQueryParameters(URI requestUri) {
        if (requestUri.getRawQuery() != null) {
            return String.format("%s?%s", requestUri.getRawPath(), requestUri.getRawQuery());
        } else {
            return requestUri.getRawPath();
        }
    }

    private String buildPathWithQueryParameters(URI requestUri) {
        if (requestUri.getRawQuery() != null) {
            return String.format("%s?%s", requestUri.getRawPath(), requestUri.getRawQuery());
        } else {
            return requestUri.getRawPath();
        }
    }
```

This code is already in the 3.x

Reply:
this would cause compilation error which did not happend while building, however checked - nothing was found